src/app/lib/config/services/hindi.js

```diff
@@ -63,14 +63,14 @@ export const service = {
       title: 'पॉडकास्ट',
       brandTitle: 'विवेचना',
       brandDescription:
-        'विवेचना: व्यक्तियों, विषयों या घटनाओं की गहरी पड़ताल का कार्यक्रम',
+        'नई रिलीज़ हुई फ़िल्मों की समीक्षा करता साप्ताहिक कार्यक्रम',
       image: {
-        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p09krvnc.jpg',
-        alt: 'विवेचना',
+        src: 'http://ichef.bbci.co.uk/images/ic/3000x3000/p09kn96g.jpg',
```

Comment:
Why are some of the links http and others are https?

Reply:
We just copied and pasted the image links provided to us :D I will change them to https, hopefully by today 🤞